stix_shifter_modules/aws_guardduty/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,2423 @@
+{
+  "FindingType": {
+    "key": "x-ibm-finding.finding_type",
+    "object": "finding"
+  },
+  "AccountId": [
+    {
+      "key": "x-aws.account_id",
```

Comment:
all the `x-aws` mappings can go under `x-aws-resource`

Reply:
Basically, x-aws-resource has been used to map the affected resources in the finding. Since account id and region are related to AWS cloud subscription and common to all findings, these fields are mapped under separate object x-aws. Kindly let us know, your comments in this.

Reply:
Updated the mappings as suggested.